nnpdf_data/nnpdf_data/commondata/CMS_SINGLETOP_13TEV/metadata.yaml

```
@@ -23,38 +23,39 @@ implemented_observables:
   npoints: []
   ndata: 1
   plotting:
-    kinematics_override: inc_sqrt_scale
+    kinematics_override: identity
     dataset_label: CMS single top $R_{t}$ 13 TeV
     plot_x: idat
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+    - zero
+    - m_t2
+    - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
+      zero:
+        description: ""
+        label: $$
+        units: ""
+      m_t2:
+        description: "top mass squared"
+        label: $M^2$
+        units: $GeV^2$
+      sqrts:
+        description: "Center of mass energy"
+        label: $\sqrt{s}$
+        units: $GeV$
     file: kinematics_TCHANNEL-XSEC.yaml
   theory:
     conversion_factor: 1.0
     operation: RATIO
     FK_tables:
     - - CMS_SINGLETOP_TCH_R_13TEV-CMS_SINGLETOP_TCH_R_13TEV_T
     - - CMS_SINGLETOP_TCH_R_13TEV-CMS_SINGLETOP_TCH_R_13TEV_TB
-  data_uncertainties: []
+  data_uncertainties: [uncertainties_TCHANNEL-XSEC.yaml]
   variants:
     legacy:
       data_uncertainties:
-      - uncertainties_legacy_TCHANNEL-XSEC.yaml
-  data_central: data_legacy_TCHANNEL-XSEC.yaml
+        - uncertainties_legacy_TCHANNEL-XSEC.yaml
+      data_central: data_legacy_TCHANNEL-XSEC.yaml
```

```suggestion
        - uncertainties_TCHANNEL-XSEC.yaml
```

Since the data and uncertainties are unchanged, please remove the old files and just make `legacy` point to the new one.

(we may remove `legacy` eventually when they are exactly equal, but one thing at a time)